openlibrary/plugins/books/tests/test_readlinks.py

```diff
@@ -7,8 +7,8 @@
 @pytest.mark.parametrize(
     "collections,subjects,options,expected",
     [
-        (['lendinglibrary'], ['Lending library'], {}, 'lendable'),
-        (['lendinglibrary'], ['Some other subject'], {}, 'restricted'),
+        (['inlibrary'], ['Lending library'], {}, 'lendable'),
+        (['inlibrary'], ['Some other subject'], {}, 'restricted'),
         (['inlibrary'], ['In library'], {}, 'restricted'),
```

Comment:
This test is now failing, but I don't understand what it is testing.

`lendinglibrary` doesn't exists as a collection.

Neither `Lending library` or `In library` appear to exist on Open Library in any quantity

https://openlibrary.org/search/subjects?q=Lending+library&mode=everything
https://openlibrary.org/search/subjects?q=In+Library&mode=everything

If they did, I'd consider them fake subjects, and suggest they be cleaned up.

Reply:
The one concrete thing I thought this was used for is setting the value of 'preview' 
in 
https://openlibrary.org/api/books.json?bibkeys=
responses.   EDIT: No, that's done in a more straighforward manner in book/dynlinks.py

Two restricted examples:
https://openlibrary.org/api/books.json?bibkeys=059035342X,0312368615

From the recently returned carousel:
https://openlibrary.org/api/books.json?bibkeys=1598220209
but it's no longer up, and the preview access states `full`?   
Full is the fallback option an doesn't capture removed items.

Unrestricted:
https://openlibrary.org/api/books.json?bibkeys=lccn:13025071
`preview: full`  seems correct.

Borrowable book:
https://openlibrary.org/api/books.json?bibkeys=1565847032
`preview: borrow`

What is the difference between dynlinks.py and readlinks.py?
both are used in openlibrary/plugins/books/code.py

Looks like readlinks.py is the 'Read API' (now called the Partner API? )
https://openlibrary.org/dev/docs/api/read

The doc page states the values for `status` are:
>'status'
>'full access', 'lendable', 'checked out' or 'restricted.' 

and the code suggests these are possible values:
https://github.com/internetarchive/openlibrary/blob/8b0f5ecdd1cd7e7088c19ac8e3e1d60eca1a95eb/openlibrary/plugins/books/readlinks.py#L259-L265